content/agent/basic_agent_usage/heroku.md

```diff
@@ -0,0 +1,94 @@
+
+The Datadog agent provides a listening port on 8125 for statsd/dogstatsd metrics and events. Traces are collected on port 8126.
+
+## Configuration
+
+In addition to the environment variables shown above, there are a number of others you can set:
+
+| Setting | Description|
+| --- | --- |
+| DD_API_KEY | *Required.* Your API key is available from the [Datadog API integrations](https://app.datadoghq.com/account/settings#api) page. Note that this is the *API* key, not the application key. |
+| DD_HOSTNAME | *Required.* Because Heroku Dynos are ephemeral and your application my be served by any available Dyno resource, you should set the hostname to your application or service name. This will give you more consistent metrics. To view metrics by Dyno hosts, the tag `dynohost` is added by the buildpack. |
+| DD_TAGS | *Optional.* Sets additional tags provided as a comma-delimited string. For example, `heroku config:set DD_TAGS=simple-tag-0,tag-key-1:tag-value-1`. The buildpack automatically adds the tags `dyno` and `dynohost` which represent the Dyno name (e.g. web.1) and host ID (e.g. ) respectively. See the ["Guide to tagging"](http://docs.datadoghq.com/guides/tagging/) for more information. |
+| DD_HISTOGRAM_PERCENTILES | *Optional.* You can optionally set additional percentiles for your histogram metrics. See [Histogram percentiles](#histogram-percentiles) below for more information. |
+| DISABLE_DATADOG_AGENT | *Optional.* When set, the Datadog agent will not be run. |
+| DD_APM_ENABLED | *Optional.* The Datadog Trace Agent (APM) will run by default. If you are not using tracing, set this to `false` to disable the Trace Agent. |
+| DD_AGENT_VERSION | *Optional.* By default, the buildpack will install the latest version of the Datadog Agent available in the package repository. Use this variable to install older versions of the Datadog Agent (note that not all versions of the Agent may be available). |
+| DD_SERVICE_NAME | *Optional.* While not read directly by the Datadog Agent, we highly recommend that you set an environment variable for your service name. See the [Service Name](#service-name) section below for more information. |
+| DD_SERVICE_ENV | *Optional.* The Datadog Agent will automatically try to identify your environment by searching for a tag in the form `env:<your environment name>`. If you do not set a tag or wish to override an existing tag, you can set the environment with this setting. For more information, see the [Datadog Tracing environments page](https://docs.datadoghq.com/tracing/environments/). |
+
+### Histogram percentiles
+
+You can optionally set additional percentiles for your histogram metrics. By default only the 95th percentile will be generated. To generate additional percentiles, set *all* percentiles, including the default, using the env variable `DD_HISTOGRAM_PERCENTILES`.  For example, if you want to generate 0.95 and 0.99 percentiles, you may use following command:
```

Comment:
r/`By default only the 95th percentile will be generated.`/`By default only the 95th percentile is generated.`